README.md

```diff
@@ -4,20 +4,18 @@ Lovely is a lua injector which embeds code into a [LÖVE 2d](https://love2d.org/
 
 ## Manual Installation
 
-**Tip:** You can navigate to the game's directory by right-clicking the game in Steam, hovering "Manage", and selecting "Browse local files".
-
 ### Windows + Proton / Wine
 
-1. Download the latest release for Windows. This will be `lovely-x86_64-pc-windows-msvc.zip`.
-2. Open the .zip archive, copy `version.dll` into the game directory.
+1. Download the latest release for Windows (located in the right sidebar under Releases). This will be `lovely-x86_64-pc-windows-msvc.zip`.
+2. Open the .zip archive, copy `version.dll` into the game directory. You can navigate to the game's directory by right-clicking the game in Steam, hovering "Manage", and selecting "Browse local files".
 3. Put one or more mods into the mod directory (NOT the same as the game directory). This should be `%AppData%/Balatro/Mods` (if you are modding Balatro).
 4. **<ins>Only Steam Deck / Proton / Wine</ins>** Set your game's launch options in Steam to `WINEDLLOVERRIDES="version=n,b" %command%`.
 5. Run the game through Steam.
 
 ### Mac
 
-1. Download the latest release for Mac. If you have an M-series CPU (M1, M2, etc.) then this will be `lovely-aarch64-apple-darwin.tar.gz`. If you have an Intel CPU then it will be `lovely-x86_64-apple-darwin.tar.gz`
-2. Open the .zip archive, copy `liblovely.dylib` and `run_lovely.sh` into the game directory.
+1. Download the latest release for Mac (located in the right sidebar under Releases). If you have an M-series CPU (M1, M2, etc.) then this will be `lovely-aarch64-apple-darwin.tar.gz`. If you have an Intel CPU then it will be `lovely-x86_64-apple-darwin.tar.gz`
+2. Open the .zip archive, copy `liblovely.dylib` and `run_lovely.sh` into the game directory. You can navigate to the game's directory by right-clicking the game in Steam, hovering "Manage", and selecting "Browse local files".
```

Comment:
Replace the "located in the right sidebar under Releases" with a link to the Releases page, https://github.com/ethangreen-dev/lovely-injector/releases